Getting-Started-With-Orleans/Grain-Persistence.md

```diff
@@ -107,6 +107,34 @@ The following attributes can be added to the `<Provider />` element to configure
 
 * __`NumStorageGrains="10"`__ (optional) - The number of grains to use to store the state, defaults to `10`
 
+### ADO.NET Persistence
+
+Initializing the database structures needs table named `Storage` from the database scripts and `OrleansQuery` with its contents (or the queries that use `Storage` table).
+The scripts are located in the Nuget library, the path is in projects close to `\packages\Microsoft.Orleans.OrleansSqlUtils.n.n.n\lib\net<version>\SQLServer\` depending on version. Deploying
+the whole script to the database should not cause problems.
+
+In addition to deploying the script to the database, the following is an example of programmatic configuration. A quick way to test this is to
```

Comment:
I don't see a reason for it to not work. The storage provider is just like any other provider, I see no benefit in treating it like a special case and add a test.